docs/README_macos_gccgfortran.txt

```diff
@@ -0,0 +1,23 @@
+#Setup instructions for macOS Mojave or Catalina using gcc-10.2.0 + gfortran-10.2.0
```

Comment:
Would it be better to add this macos file in a separate PR at the same time that you merge the upcoming macos PR into regional_workflow?

Reply:
The macos build additions were added to the release branch a while ago (#62), so this just brings the READMEs up-to-date with the release branch. I don't think it's worth omitting.